PerimeterXModule/PxModule.cs

```diff
@@ -421,13 +452,14 @@ private static string ByteArrayToHexString(byte[] input)
 
         private void HandleVerification(HttpApplication application)
         {
-            bool verified = blockingScore > pxContext.Score;
             PxModuleConfigurationSection config = (PxModuleConfigurationSection)ConfigurationManager.GetSection(PxConstants.CONFIG_SECTION);
+			bool verified = blockingScore > pxContext.Score || config.MonitorMode;
 
-            Debug.WriteLine(string.Format("Request score: {0}, blocking score: {1}, monitor mode status: {2}.", pxContext.Score, blockingScore, config.MonitorMode == true ? "On" : "Off"), PxConstants.LOG_CATEGORY);
+			Debug.WriteLine(string.Format("Request score: {0}, blocking score: {1}, monitor mode status: {2}.", pxContext.Score, blockingScore, config.MonitorMode == true ? "On" : "Off"), PxConstants.LOG_CATEGORY);
```

Comment:
Please apply automatic formatting to replace tabs with (4) spaces on all of these files (there are many such places with weird spacing). You can configure it on your Mac Visual Studio (like we did on mine) and then use the shortcut ctrl+shift+f.